### PR TITLE
Update the list of api endpoints for curve pools

### PR DESCRIPTION
**Pull request by yabirgb**
Asked in telegram and got a list with the endpoints missing https://github.com/curvefi/curve-api/pull/323/

The diff on the globaldb is https://gist.github.com/yabirgb/bd9a4c5c1d91b9d3a397960dc49061c4

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/curve/curve_cache.py [hunk 1]
```diff
@@ -50,12 +50,15 @@
 # list of pools that we know contain bad tokens
 IGNORED_CURVE_POOLS = {'0x066B6e1E93FA7dcd3F0Eb7f8baC7D5A747CE0BF9'}
 
-CURVE_API_URLS = [
+CURVE_API_URLS = (
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/curve/curve_cache.py:53]: Does this mean that queryign the trylsd pool in that test I made for curve pool price would work now? Can you do it? I wanted to have a 3 token pool and a 4 token pool test in there but could not due to those missing.

Would it mean that it needs no more changes to the global DB as they are pulled directly from the API?

**Review by LefterisJP — COMMENTED** [review]
Can you show the global DB diff somewhere.

At least one of the tests failing is due to the assets PR we merged. I forgot that these tests needed adjustment. I will look at them in another branch.

**Comment by LefterisJP** [comment]
I think you also added all assets from update 23 which should not be done yet. The zksync lite assets I have not added to the global DB yet. This is why the failures. In normal develop those tests do not fail.

File: rotkehlchen/tests/unit/decoders/test_eigenlayer.py [hunk 2]
```diff
@@ -36,14 +38,26 @@ def test_deposit_token(database, ethereum_inquirer, ethereum_accounts):
         location_label=ethereum_accounts[0],
         notes=f'Burned {gas_amount} ETH for gas',
         counterparty=CPT_GAS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=112,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.APPROVE,
```
Comment (yabirgb) [inline rotkehlchen/tests/unit/decoders/test_eigenlayer.py:47]: Explained here https://github.com/rotki/rotki/pull/7538/commits/6ca040fd2c2d8e7c50f2bbbb718ffb2f53b6f491 why it was missing

**Review by LefterisJP — COMMENTED** [review]
1. For some reason a velodrome test failed. Can you check it?
2. We will need the exact addition/script to populate the global DB with since this is bugfixes and will need a merge in develop soon, and at that merge we will need to:
    a. Revert/choose develop's global DB
    b. Add to it the aforementioned change.

**Comment by yabirgb** [comment]
Here it is the script that can be used

https://gist.github.com/yabirgb/7280386094eba307de2b7795d31f322b

tested it with 

```
sqlite3 data/base_global.db -init miau.sql .quit
```

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**